testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/messaging/CoreBridgeCallTimeoutTestCase.java

```diff
@@ -0,0 +1,220 @@
+import javax.jms.Session;
+import javax.jms.TextMessage;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import java.util.Collections;
+import java.util.Properties;
+import java.util.UUID;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * This test verifies that setting a call timeout on a core bridge takes an effect.
+ *
+ * Scenario:
+ *
+ * <ol>
+ * <li>Create a bridge without a call-timeout.</li>
+ * <li>Verify that messages are transferred.</li>
+ * <li>Introduce a delay to bridge calls via a byteman rule and recreate bridge with call-timeout value of 500 ms.</li>
+ * <li>Verify that messages are not transferred.</li>
+ * </ol>
+ *
+ * @author Tomas Hofman
+ */
+@RunWith(Arquillian.class)
+@RunAsClient
+public class CoreBridgeCallTimeoutTestCase {
+
+    private static final Logger logger = Logger.getLogger(CoreBridgeCallTimeoutTestCase.class);
+
+    private static final String CONTAINER_QUALIFIER = "default-full-jbossas-byteman";
+    private static final String EXPORTED_PREFIX = "java:jboss/exported/";
+    private static final String REMOTE_CONNECTION_FACTORY_LOOKUP = "jms/RemoteConnectionFactory";
+    private static final String SOURCE_QUEUE_NAME = "SourceQueue";
+    private static final String SOURCE_QUEUE_LOOKUP = "jms/SourceQueue";
+    private static final String TARGET_QUEUE_NAME = "TargetQueue";
+    private static final String TARGET_QUEUE_LOOKUP = "jms/TargetQueue";
+    private static final String BRIDGE_NAME = "CoreBridge";
+    private static final String MESSAGE_TEXT = "Hello world!";
+
+    private static final Submit BYTEMAN = new Submit(
+            System.getProperty("byteman.server.ipaddress", Submit.DEFAULT_ADDRESS),
+            Integer.getInteger("byteman.server.port", Submit.DEFAULT_PORT));
+
+    @ArquillianResource
+    protected static ContainerController container;
+
+    private JMSOperations jmsOperations;
+    private ManagementClient managementClient;
+
+    @Before
+    public void setUp() {
+        if (!container.isStarted(CONTAINER_QUALIFIER)) {
+            container.start(CONTAINER_QUALIFIER);
+        }
+
+        managementClient = createManagementClient();
+        jmsOperations = JMSOperationsProvider.getInstance(managementClient.getControllerClient());
+
+        // create source and target queues and a bridge
+        jmsOperations.createJmsQueue(SOURCE_QUEUE_NAME, EXPORTED_PREFIX + SOURCE_QUEUE_LOOKUP);
+        jmsOperations.createJmsQueue(TARGET_QUEUE_NAME, EXPORTED_PREFIX + TARGET_QUEUE_LOOKUP);
+        createCoreBridge(null);
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        // clean up
+        jmsOperations.removeJmsQueue(SOURCE_QUEUE_NAME);
+        jmsOperations.removeJmsQueue(TARGET_QUEUE_NAME);
+        jmsOperations.removeCoreBridge(BRIDGE_NAME);
```

Comment:
you need to close the management client

Reply:
and the jmsOperations (even if this does nothing for now)

Reply:
Closed both in `tearDown()`.